### DISCUSSION TITLE
Resources to use Weights and Biases with MMDetection

### DISCUSSION THREAD

**Discussion by ayulockin** [open-mmlab/mmdetection#9170] [Show and tell]
Hey good folks at MMDetection now that we have a dedicated `MMDetWandbHook` for MMDetection, we have some resources that we think would be helpful for MMDetection users.

## Colab Notebooks

🧨 [Train an Object Detection + Semantic Segmentation Model with MMDetection and W&B](https://github.com/wandb/examples/blob/master/colabs/mmdetection/Train_an_Object_Detection%2BSemantic_Segmentation_Model_with_MMDetection_and_W%26B.ipynb)

🧯 [Train Object Detector with MMDetection and W&B](https://github.com/wandb/examples/blob/master/colabs/mmdetection/Train_Object_Detector_with_MMDetection_and_W%26B.ipynb)

## Article (W&B Report)

If you don't want to run the colab notebooks, read our W&B report on `MMDetWandbHook`. You can also interact with the logged metrics, model predictions, system metrics, etc.

🔥 [How to Use Weights & Biases with MMDetection?](https://wandb.ai/ayush-thakur/mmdetection/reports/How-to-Use-Weights-Biases-with-MMDetection--VmlldzoyMTM0MDE2)

We also have the same report in [Chinese](https://wandb.ai/wandb_fc/chinese/reports/-MMDetection-Weights-Biases--VmlldzoyMTg3MjQ0), [Japanese](https://wandb.ai/wandb_fc/japanese/reports/MMDetection-Weights-Biases---VmlldzoyMTg3MjQ1), and [Korean](https://wandb.ai/wandb_fc/korean/reports/MMDetection-Weights-Biases---VmlldzoyMjA1Njky).

## Documentation

The users can find the documentation both in English and Chinese. 

🧑‍🚒 Docs in English: https://docs.wandb.ai/guides/integrations/mmdetection

🚒 Docs in Chinese: https://docs.wandb.ai/v/zh-hans/integrations/mmdetection

## Let's spread the word

We at Weights and Biases would love to spread the usefulness of MMDetection and the OpenMMLabs ecosystem with ML communities. If you think we can collaborate more closely would love it, if you can reach out to ayusht@wandb.com.

Tagging @jbwang1997, @hhaAndroid, @ZwwWayne, @RangiLyu, and Julled since you all reviewed #7459 that added `MMDetWandbHook`.

**Comment by jbwang1997** [comment]
Thank you very much for the tutorial. It is indeed helpful for users to familiarize themselves with the feature of MMDetWandbHook. We will post the link in our tutorials. Looking forward to working with you next time.👍

**Comment by ayulockin** [comment]
Thanks @jbwang1997, posting the resources in your tutorials would be lovely. 

Looking forward to more collaborations. :D
